.github/workflows/ros1.yml

```diff
@@ -41,17 +41,19 @@ jobs:
       run: |
         # TODO(ros-tooling/setup-ros-docker#7): calling chown is necessary for now
         sudo chown -R rosbuild:rosbuild "$HOME" .
+    - name: Scan using git-secrets
+      uses: aws-robotics/aws-robomaker-github-actions/git-secrets-scan-action@2.4.0
```

Comment:
You had a concern that doing `--scan-history` would take a long time, right?

Anyway, I think we should move the scan to a separate workflow YAML file, so that the scanning can take place in parallel with building and bundling.

Reply:
Yes I did.. but I tested --scan-history on some public repos with larger commit history and the time doesn't seem to exceed more than a minute at the most. I feel that should be considerably lesser than what build and bundle takes.

Reply:
Yes I did.. but I tested --scan-history on some public repos with signicantly larger commit history and the time doesn't seem to exceed more than a minute at the most. I feel that should be considerably lesser than what build and bundle takes.